wpilib/wpilib/_impl/main.py

```diff
@@ -11,6 +12,31 @@
 from .logconfig import configure_logging
 import hal_impl
 
+# Make silly mistakes more obvious
+_show_run_warning = True
+
+
+def _atexit():
+    if _show_run_warning and not "pytest" in sys.modules:
```

Comment:
lol. perhaps it'd be a better idea to move the pytest check down to the atexit register call?

Reply:
Eh.

Reply:
Wouldn't this have a not-good impact on the time to run the tests though? Like, we have almost 900 tests, and wpilib gets reloaded for every one of those.